### PR TITLE
Better Logging

### PR DESCRIPTION
**Pull request by Kayanski**
This PR aims at enhancing the logging of cw-orch by providing different styles when logging different information.

This also allows to show to users that logs are available and that they should enable logging capabilities.

Closes ORC-30

## Env variable classification proposition

- Local configuration (env variables, state file, path) --> Should we move that to debug ?
- Endpoint connection (gRPC + Account identification) `CONNECTIVITY_LOGS`
- Contract Interactions `CONTRACT_LOGS`
- Sending transactions + Receiving transactions responses `TRANSACTION_LOGS`
- Transaction sending retries `TX_RETRY_LOGS`

Here is an example log today : 

image

### CODE REVIEW

File: cw-orch-daemon/src/log.rs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+pub fn print_if_disabled_log() {
+    // Here we check for logging capabilites. We want to help the users as much as possible
+
+    if !log::log_enabled!(log::Level::Info) {
+        println!(
+            "It seems like you haven't enabled logs. In order to do so, you have to : 
+    - use `env_logger::init()` inside your script
+    - use the env variable `RUST_LOG=INFO` for standard logs or `RUST_LOG=DEBUG` for more details"
+        )
+    }
```
Comment (Buckram123) [inline cw-orch-daemon/src/log.rs:11]: Will it warn in case level `Debug` provided?

Comment (Kayanski) [reply cw-orch-daemon/src/log.rs:11]: No, the info level is only a minimum

File: cw-orch-daemon/src/queriers/node.rs [hunk 2]
```diff
@@ -233,14 +234,14 @@ impl Node {
...
-                    log::debug!("TX found: {:?}", resp);
+                    log::debug!(target: QUERY_LOGS, "TX found: {:?}", resp);
                     return Ok(resp.into());
                 }
                 Err(err) => {
                     // increase wait time
                     block_speed = (block_speed as f64 * 1.6) as u64;
-                    log::debug!("TX not found with error: {:?}", err);
-                    log::debug!("Waiting {block_speed} seconds");
+                    log::debug!(target: QUERY_LOGS, "TX not found with error: {:?}", err);
```
Comment (CyberHoward) [inline cw-orch-daemon/src/queriers/node.rs:243]: warn?

File: cw-orch-daemon/src/sync/builder.rs [hunk 3]
```diff
@@ -73,6 +73,8 @@ impl DaemonBuilder {
         // build the underlying daemon
         let daemon = rt_handle.block_on(DaemonAsyncBuilder::from(self.clone()).build())?;
 
+        print_if_disabled_log();
+
```
Comment (CyberHoward) [inline cw-orch-daemon/src/sync/builder.rs:77]: This is already called when building the async daemon.

**Review by CyberHoward — APPROVED** [review]

**Pull request merged**